assets/cubyz/biomes/trans/beach.zig.zon

```diff
@@ -0,0 +1,33 @@
+.{
+	.properties = .{
+		.ocean,
+		.wet,
+	},
+	.radius = 128,
+	.minHeight = 2,
+	.maxHeight = 12,
+	.chance = 0,
+	.roughness = 1,
+
+	.interpolation = .linear,
+	
+	.hills = 2,
+
+	.music = "cubyz:tides",
+
+		.structures = .{
+		.{
+			.id = "cubyz:ground_patch",
+			.block = "cubyz:sand",
+			.chance = 1,
+			.width = 4,
+			.variation = 8,
+			.depth = 4,
+			.smoothness = 0.8,
+		},
```

Comment:
Could you explain why you want to have ground patches that fill the entire biome?

Reply:
Yeah, but then we get trees that grow on sand and such.
The biome transition should probably be improved in the code instead.